lib/rubocop/cop/rails/squished_sql_heredocs.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      #
+      # Checks multi- and singleline SQL heredocs to use `.squish`.
```

Comment:
The cop doesn't make any distinction between multi- and single-line heredocs any more, do we need to mention them here?